packages/authentication/src/definitions.ts

```diff
@@ -1613,7 +1613,7 @@ export interface ActionCodeSettings {
    * When multiple custom dynamic link domains are defined for a project, specify which one to use
    * when the link is to be opened via a specified mobile app (for example, `example.page.link`).
    */
-  dynamicLinkDomain?: string;
+  linkDomain?: string;
```

Comment:
This would be a breaking change. Add `linkDomain` as a new property and add a deprecation notice to the `dynamicLinkDomain` property.